.gitignore

```diff
@@ -3,8 +3,6 @@
 *.code-workspace
```

Comment:
Looking at the gitignore, in line 1 there's extra space after `VSCode` that's going to bother me unless it gets removed, can you just fix it in this PR